stripper/ze_m0w0m_csgo1.cfg

```diff
@@ -84,3 +84,71 @@ modify:
 		"health" "4"
 	}
 }
+modify:
+{
+	match:
+	{
+		"classname" "ambient_generic"
+		"targetname" "chekebo"
+	}
+	replace:
+	{
+		"health" "3"
+	}
+}
+
+;Make Sphere Freeze Ifrit
+modify:
+{
+	match:
+	{
+		"classname" "logic_relay"
+		"targetname" "poison_relay"
+	}
+	insert:
+	{
+		"OnTrigger" "bosslvl3_phy,DisableMotion,,0,-1"
+		"OnTrigger" "bosslvl3_phy,EnableMotion,,6,-1"
+	}
+}
+
+;Nerf Golem and Ifrit HP
+modify:
+{
+	match:
+	{
+		"classname" "func_physbox"
+		"targetname" "bosslvl3_hit"
+	}
+	delete:
+	{
+		"OnUser1" "!selfRunScriptCodeAddHealth(30);01"
+		"OnUser2" "!selfRunScriptCodeAddHealth(20);01"
+		"OnUser3" "!selfRunScriptCodeAddHealth(42);01"
+	}
+	insert:
+	{
+		"OnUser1" "!selfRunScriptCodeAddHealth(20);01"
+		"OnUser2" "!selfRunScriptCodeAddHealth(10);01"
+		"OnUser3" "!selfRunScriptCodeAddHealth(30);01"
+	}
+}
+
+modify:
+{
+	match:
+	{
+		"classname" "func_physbox"
+		"targetname" "bosslvl3_golem_hit"
+	}
+	delete:
+	{
+		"OnUser1" "!selfRunScriptCodeAddHealth(20);01"
+		"OnUser2" "!selfRunScriptCodeAddHealth(28);01"
+	}
+	insert:
+	{
+		"OnUser1" "!selfRunScriptCodeAddHealth(8);01"
+		"OnUser2" "!selfRunScriptCodeAddHealth(12);01"
+	}
+}
```

Comment:
I don't recall there being any noticeable balancing issues on this boss during testing. Additionally, it's a tad early to be doing significant balance changes to a map we just added to the server anyways. Unless something was horribly balanced.